### DISCUSSION TITLE
「Markdown記法 チートシート」から「見出し」の「結果」が消えた

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#597] [要望・アイデア・不具合]
[Markdown記法 チートシート](https://qiita.com/Qiita/items/c686397e4a0f4f11683d) の「Headings - 見出し」の項には、「書き方」はありますが「結果」が示されていません。

以前には「結果」も明示されていたように記憶していますが、意図的な削除ですか？（想定する利用者層を初級から中級に変えたとか）　それともミスによる誤削除ですか？　編集履歴を見ても、その辺の事情は良く分かりません。

自分の初投稿の頃を振り返ってみれば、「結果」も明示されている方が有難く感じます。

**Comment by tanaka-asami415** [comment]
@tsubolabo 
ご報告ありがとうございます🙏

確認いたしましたが「Headings - 見出し」については、最初から「結果」の記載をしておりませんでした。
「結果」も記載がある方がいいと思いますので、追記いたします！

**Reply by tanaka-asami415** [reply]
@tsubolabo 
追記いたしました！✨
[Markdown記法 チートシート](https://qiita.com/Qiita/items/c686397e4a0f4f11683d#headings---%E8%A6%8B%E5%87%BA%E3%81%97)
